Foundation/include/Poco/ArchiveStrategy.h

```diff
@@ -89,7 +89,7 @@ class ArchiveByTimestampStrategy: public ArchiveStrategy
 	{
 	}
 	
-	LogFile* archive(LogFile* pFile)
+	virtual LogFile* archive(LogFile* pFile)
```

Comment:
`archive` in the base class (`ArchiveStrategy`) is already declared as virtual, so we don't need to add a virtual here

https://github.com/pingcap/poco/blob/354379ae4253f9a6805e8ba8f051c7d8497b4adc/Foundation/include/Poco/ArchiveStrategy.h#L45-L48